gui/wxpython/startup/guiutils.py

```diff
@@ -279,44 +280,66 @@ def rename_location_interactively(guiparent, grassdb, location):
     return newlocation
 
 
-def delete_mapset_interactively(guiparent, grassdb, location, mapset):
+def delete_mapset_interactively(guiparent, db_loc_mapset):
     """
-    Delete selected mapset
+    Delete selected mapset/s. Whether PERMANENT or current mapsets found,
+    delete operation is not performed.
     """
-    if mapset == "PERMANENT":
-        GMessage(
+    genv = gisenv()
+
+    # Initialization
+    issues = []
+
+    # Check selected mapsets and remember issues
+    for grassdb, location, mapset in db_loc_mapset:
+
+        # Check for permanent mapsets
+        if mapset == "PERMANENT":
+            issues.append(os.path.join(grassdb, location, mapset) +
```

Comment:
Use format function as we discussed everywhere.

Reply:
Changed in the whole guiutils.py file.